### DISCUSSION TITLE
HyprlandTaskbar -> Grouped Icons Mode

### DISCUSSION THREAD

**Discussion by KKonaOG** [nwg-piotr/nwg-shell#464] [Ideas]
I'm mostly writing this as I have written a custom version to include this functionality and was interested in knowing if making a PR would be worthwhile.

Grouped Icons Mode would convert HyprlandTaskbars into something similar to a traditional taskbar menu. 

See images below:

![grouped_screenshot1](https://github.com/user-attachments/assets/4c90260e-c958-4629-88de-e3a24fbc77db)

![grouped_screenshot2(1)](https://github.com/user-attachments/assets/d5669a5b-958d-4db7-941b-326e439d8652)

It has most common features but does not support pinning or re-arrangement at this point in time. Order of the icons is relatively arbitrary.

**Comment by nwg-piotr** [comment]
Why not, provided that it doesn't brake existing features.
